specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/definitions.json

```diff
@@ -233,6 +233,29 @@
       },
       "description": "Paged API list representation."
     },
+    "DocumentationCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
```

Comment:
>array [](http://example.com/codeflow?start=19&length=5)

Recommended to use "x-ms-identifiers" in all array, to put identifier of items in that array. See: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-identifiers](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-identifiers).